src/lib/realmdb/RealmDB.js

```diff
@@ -21,19 +21,20 @@ import NewsSource from './feedSource/NewsSource'
 import TransactionsSource from './feedSource/TransactionsSource'
 import { makeCategoryMatcher } from './feed'
 
-// when 'failed to fetch' increase delay before next try to 5 seconds
+// when 'failed to fetch' increase delay before next try for 1.5x times
 const _retryMiddleware = (exception, options, defaultOptions) => {
+  const { interval } = options
   const { message } = exception || {}
 
-  // if not failed to fetch - reset interval to the default one
+  // if not 'failed to fetch' - reset interval to the default one
   if (!(message || '').startsWith('Failed to fetch')) {
     return defaultOptions
   }
 
-  // otherwise increase delay interval to 5 seconds
+  // otherwise increase delay interval 1.5x from current value
   return {
     ...options,
-    interval: 5000,
+    interval: (3 * interval) >> 1,
```

Comment:
@serdiukov-o-nordwhale 
why are you using magic tricks written in Chinese? ask your team if they can explain it to you, if not then please write it in human style. there are other ways to make it an Integer, yes it is a little longer, but understandable

Reply:
1. `>> 2` is shorter and works faster than `Math.floor(something / 2)`. 
2. bit shift operator is one of the basic operators in **any** programming language. it's well understandable by developers like other operators such as math or logic ones

Reply:
@sirpy i've lost a few messages because you linked my old account @serdiukov-o-nordwhale 
Please tag me as @johnsmith-gooddollar so i would be able to receive notifications. thanks

Reply:
@johnsmith-gooddollar I know exactly what it does, I didnt ask for code explanation, it was a rhetorical question
by magic tricks I mean low level code that most developers do not understand and is not suitable when using high level languages that do not deal with extreme efficiency.
what exactly are you saving here? 1 nano second?
we've been over this 1000 times, NO MICRO OPTIMIZATIONS, write code that is readable.

Reply:
i just wanted to do integer division using a single operator.
some languages has dedicated int div operator `/` but JS doesn't
but if you divide by the `2^n` you could use bit shift `>> n`. 

but ok, i've changed: https://github.com/GoodDollar/GoodDAPP/commit/c2dd2417add3e796833d11184d50378d748a0c84